src/Components/GEOSldas_GridComp/GEOSlandpert_GridComp/LDAS_PertRoutines.F90

```diff
@@ -1111,6 +1113,8 @@ subroutine get_force_pert_inputs( pert_grid_l,                         &
        call MPI_BCAST(stdfilename_force_pert,300,MPI_CHARACTER,0,mpicomm,mpierr)
 
        nc4_file = stdfilename_force_pert
+       ! NOTE: the input file is in netcdf, with a group 'std_force_pert',
+       ! and the grid is same as the *global* grid (tile_grid_g) (sqz 2/2023) 
```

Comment:
@saraqzhang, I'm a bit confused by this statement.  In line 661 above, there's a comment that when the model is on the cube-sphere tile space, the pert grid is a hard-wired lat/lon grid (with approximately the same resolution).  The comment here (lines 1116-1117) and again below (lines 1510-1511) suggest that the pert grid is always tile_grid_g.  But for the cube-sphere tile space, this isn't correct.  Maybe I'm misunderstanding this?

Reply:
The tile_grid_g here is already a lat-lon grid generated here  https://github.com/GEOS-ESM/GEOSldas/blob/eef59e2d1b69ce8a91122cbbb4a641bdecc810f6/src/Components/GEOSldas_GridComp/GEOS_LdasGridComp.F90#L632

Reply:
this comment should be more clear:   the grid in netcdf file is lat-lon grid, the resolution should match what "get_pert_grid(tile_grid_g, latlon_tmp_g) "   generated.   
it is left to user to find out what the resolution should be.  ( for instance in a C180 run, it is 720x540)